[2013-08-27 06:13:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:13:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:13:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:19:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(302): Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 [internal function]: Illuminate\Routing\Controllers\Controller->__call('show', Array)
#2 [internal function]: UserController->show('addTestUser')
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#6 [internal function]: Illuminate\Routing\{closure}('addTestUser')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 06:19:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(302): Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 [internal function]: Illuminate\Routing\Controllers\Controller->__call('show', Array)
#2 [internal function]: UserController->show('addTestUser')
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#6 [internal function]: Illuminate\Routing\{closure}('addTestUser')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 06:19:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(302): Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 [internal function]: Illuminate\Routing\Controllers\Controller->__call('index', Array)
#2 [internal function]: UserController->index()
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#6 [internal function]: Illuminate\Routing\{closure}()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 06:20:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(302): Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 [internal function]: Illuminate\Routing\Controllers\Controller->__call('show', Array)
#2 [internal function]: UserController->show('addTestUser')
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#6 [internal function]: Illuminate\Routing\{closure}('addTestUser')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 06:22:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:22:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:22:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:22:46] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:22:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:22:53] log.ERROR: exception 'ErrorException' with message 'hash() expects at least 2 parameters, 1 given' in D:\wamp\www\compod\compod\app\controllers\UserController.php:60
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'hash() expects ...', 'D:\wamp\www\com...', 60, Array)
#1 D:\wamp\www\compod\compod\app\controllers\UserController.php(60): hash('test')
#2 [internal function]: UserController->getAddTestUser()
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getAddTestUser', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getAddTestUser', Array)
#6 [internal function]: Illuminate\Routing\{closure}()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 06:23:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 06:23:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 06:23:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 06:23:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 06:23:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:09:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:09:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:09:28] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:09:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:09:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:09:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:10:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:10:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:10:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:10:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:10:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php:290
Stack trace:
#0 [internal function]: Illuminate\Routing\Controllers\Controller->missingMethod(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('missingMethod', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'missingMethod', Array)
#4 [internal function]: Illuminate\Routing\{closure}(Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#11 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#12 {main} [] []
[2013-08-27 10:11:36] log.ERROR: exception 'ErrorException' with message 'hash() expects at least 2 parameters, 1 given' in D:\wamp\www\compod\compod\app\controllers\UserController.php:59
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'hash() expects ...', 'D:\wamp\www\com...', 59, Array)
#1 D:\wamp\www\compod\compod\app\controllers\UserController.php(59): hash('test')
#2 [internal function]: UserController->getAddtestuser()
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getAddtestuser', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getAddtestuser', Array)
#6 [internal function]: Illuminate\Routing\{closure}()
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 10:13:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:13:33] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:15:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:15:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:15:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:15:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:15:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:17:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 10:17:21] log.ERROR: exception 'PDOException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bert@demax.be' for key 'users_email_unique'' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php:330
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(330): PDOStatement->execute(Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(520): Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'insert into `us...', Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(331): Illuminate\Database\Connection->run('insert into `us...', Array, Object(Closure))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(288): Illuminate\Database\Connection->statement('insert into `us...', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `us...', Array)
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1380): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `us...', Array, 'id')
#6 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(731): call_user_func_array(Array, Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1185): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1185): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1155): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1067): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#12 D:\wamp\www\compod\compod\app\controllers\UserController.php(60): Illuminate\Database\Eloquent\Model->save()
#13 [internal function]: UserController->getAddtestuser()
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#15 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getAddtestuser', Array)
#16 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getAddtestuser', Array)
#17 [internal function]: Illuminate\Routing\{closure}()
#18 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#19 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#20 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#22 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#23 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#24 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#25 {main}

Next exception 'Exception' with message 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bert@demax.be' for key 'users_email_unique' (SQL: insert into `users` (`username`, `email`, `active`, `password`, `updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?)) (Bindings: array (
  0 => 'Karel',
  1 => 'bert@demax.be',
  2 => 1,
  3 => 'test',
  4 => '2013-08-27 10:17:21',
  5 => '2013-08-27 10:17:21',
))' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php:555
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(528): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'insert into `us...', Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(331): Illuminate\Database\Connection->run('insert into `us...', Array, Object(Closure))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Connection.php(288): Illuminate\Database\Connection->statement('insert into `us...', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `us...', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1380): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `us...', Array, 'id')
#5 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(731): call_user_func_array(Array, Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1185): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1185): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1155): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1067): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#11 D:\wamp\www\compod\compod\app\controllers\UserController.php(60): Illuminate\Database\Eloquent\Model->save()
#12 [internal function]: UserController->getAddtestuser()
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getAddtestuser', Array)
#15 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getAddtestuser', Array)
#16 [internal function]: Illuminate\Routing\{closure}()
#17 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#18 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#19 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#20 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#21 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#22 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#23 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#24 {main} [] []
[2013-08-27 10:18:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:20:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:20:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:21:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:26:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:26:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:27:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:33:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:35:12] log.ERROR: exception 'InvalidArgumentException' with message 'Driver [fluent] not supported.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php:82
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(15): Illuminate\Support\Manager->createDriver('fluent')
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php(54): Illuminate\Auth\AuthManager->createDriver('fluent')
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php(127): Illuminate\Support\Manager->driver()
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Illuminate\Support\Manager->__call('check', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Illuminate\Auth\AuthManager->check()
#5 D:\wamp\www\compod\compod\app\filters.php(60): Illuminate\Support\Facades\Facade::__callStatic('check', Array)
#6 D:\wamp\www\compod\compod\app\filters.php(60): Illuminate\Support\Facades\Auth::check()
#7 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(159): call_user_func_array(Object(Closure), Array)
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(100): Illuminate\Routing\Route->callFilter('guest', Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(43): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#15 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#16 {main} [] []
[2013-08-27 18:35:44] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Html\FormBuilder::open() must be an array, string given, called in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 177 and defined' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php:100
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(100): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\wamp\www\com...', 100, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open('login')
#2 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#3 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open('login')
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#9 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#16 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#17 {main} [] []
[2013-08-27 18:38:33] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Html\FormBuilder::open() must be an array, string given, called in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 177 and defined' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php:100
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(100): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\wamp\www\com...', 100, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open('login')
#2 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#3 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open('login')
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#9 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#16 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#17 {main} [] []
[2013-08-27 18:38:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:38:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 18:49:23] log.ERROR: exception 'ErrorException' with message 'include(D:\wamp\www\compod\compod/app/controllers/BaseController.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory' in D:\wamp\www\compod\compod\vendor\composer\ClassLoader.php:185
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\composer\ClassLoader.php(185): Illuminate\Exception\Handler->handleError(2, 'include(D:\wamp...', 'D:\wamp\www\com...', 185, Array)
#1 D:\wamp\www\compod\compod\vendor\composer\ClassLoader.php(185): Composer\Autoload\ClassLoader::loadClass()
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('BaseController')
#3 D:\wamp\www\compod\compod\app\controllers\UserController.php(3): spl_autoload_call('BaseController')
#4 D:\wamp\www\compod\compod\vendor\composer\ClassLoader.php(185): include('D:\wamp\www\com...')
#5 [internal function]: Composer\Autoload\ClassLoader->loadClass('UserController')
#6 [internal function]: spl_autoload_call('UserController')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Container\Container.php(296): ReflectionClass->__construct('UserController')
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Container\Container.php(238): Illuminate\Container\Container->build('UserController', Array)
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(434): Illuminate\Container\Container->make('UserController', Array)
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('UserController')
#11 [internal function]: Illuminate\Routing\{closure}()
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#18 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#19 {main} [] []
[2013-08-27 19:05:35] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Html\FormBuilder::open() must be an array, string given, called in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 177 and defined' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php:100
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(100): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\wamp\www\com...', 100, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open('login')
#2 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#3 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open('login')
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#9 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#16 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#17 {main} [] []
[2013-08-27 19:10:51] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Html\FormBuilder::open() must be an array, string given, called in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 177 and defined' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php:100
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(100): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\wamp\www\com...', 100, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open('login')
#2 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#3 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open('login')
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#9 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#16 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#17 {main} [] []
[2013-08-27 19:10:55] log.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Illuminate\Html\FormBuilder::open() must be an array, string given, called in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php on line 177 and defined' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php:100
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(100): Illuminate\Exception\Handler->handleError(4096, 'Argument 1 pass...', 'D:\wamp\www\com...', 100, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open('login')
#2 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#3 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open('login')
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#9 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#16 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#17 {main} [] []
[2013-08-27 19:11:35] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '=', expecting ')'' in D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e:1
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-27 19:12:04] log.ERROR: exception 'InvalidArgumentException' with message 'Unknown action [login].' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php:263
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(812): Illuminate\Routing\UrlGenerator->action('login')
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(761): Illuminate\Html\FormBuilder->getControllerAction('login')
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Html\FormBuilder.php(109): Illuminate\Html\FormBuilder->getAction(Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Html\FormBuilder->open(Array)
#4 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Facade::__callStatic('open', Array)
#5 D:\wamp\www\compod\compod\app\storage\views\05a8bd3494fc89d161da39cea1ca012e(1): Illuminate\Support\Facades\Form::open(Array)
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#11 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#13 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#14 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#18 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#19 {main} [] []
[2013-08-27 19:16:46] log.ERROR: exception 'InvalidArgumentException' with message 'Driver [fluent] not supported.' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php:82
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php(15): Illuminate\Support\Manager->createDriver('fluent')
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php(54): Illuminate\Auth\AuthManager->createDriver('fluent')
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Manager.php(127): Illuminate\Support\Manager->driver()
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Support\Manager->__call('attempt', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Auth\AuthManager->attempt(Array)
#5 D:\wamp\www\compod\compod\app\routes.php(27): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#6 D:\wamp\www\compod\compod\app\routes.php(27): Illuminate\Support\Facades\Auth::attempt(Array)
#7 [internal function]: {closure}()
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#14 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#15 {main} [] []
[2013-08-27 19:17:47] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\wamp\www\compod\compod\app\routes.php:31
Stack trace:
#0 D:\wamp\www\compod\compod\app\routes.php(31): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\wamp\www\com...', 31, Array)
#1 [internal function]: {closure}()
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#8 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#9 {main} [] []
[2013-08-27 19:19:54] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\wamp\www\compod\compod\app\routes.php:31
Stack trace:
#0 D:\wamp\www\compod\compod\app\routes.php(31): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\wamp\www\com...', 31, Array)
#1 [internal function]: {closure}()
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#8 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#9 {main} [] []
[2013-08-27 19:24:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#5 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#6 {main} [] []
[2013-08-27 19:31:27] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\wamp\www\compod\compod\app\storage\views\9ed37656c939add03ff35863e8ef134d:2
Stack trace:
#0 D:\wamp\www\compod\compod\app\storage\views\9ed37656c939add03ff35863e8ef134d(2): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\wamp\www\com...', 2, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 19:31:27] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\wamp\www\compod\compod\app\storage\views\9ed37656c939add03ff35863e8ef134d:2
Stack trace:
#0 D:\wamp\www\compod\compod\app\storage\views\9ed37656c939add03ff35863e8ef134d(2): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\wamp\www\com...', 2, Array)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\wamp\www\com...')
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\wamp\www\com...', Array)
#3 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\wamp\www\com...', Array)
#4 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\wamp\www\compod\compod\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
[2013-08-27 19:39:16] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in D:\wamp\www\compod\compod\app\routes.php:50
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-08-27 19:39:52] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "login" as such route does not exist.' in D:\wamp\www\compod\compod\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php:134
Stack trace:
#0 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('login', Array, true)
#1 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Redirector.php(144): Illuminate\Routing\UrlGenerator->route('login', Array)
#2 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(177): Illuminate\Routing\Redirector->route('login')
#3 D:\wamp\www\compod\compod\app\filters.php(66): Illuminate\Support\Facades\Facade::__callStatic('route', Array)
#4 D:\wamp\www\compod\compod\app\filters.php(66): Illuminate\Support\Facades\Redirect::route('login')
#5 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#6 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(159): call_user_func_array(Object(Closure), Array)
#7 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(100): Illuminate\Routing\Route->callFilter('auth', Object(Illuminate\Http\Request))
#8 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Route.php(43): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#9 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\wamp\www\compod\compod\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\wamp\www\compod\compod\public\index.php(49): Illuminate\Foundation\Application->run()
#13 D:\wamp\www\compod\compod\server.php(19): require_once('D:\wamp\www\com...')
#14 {main} [] []
